scripts/Docker/readme.md

```diff
@@ -0,0 +1,76 @@
+# Librealsense Docker
+
+## Pre-Work: Docker Installation
+Install docker in the environemnt using the  following tutorial (adjust the OS):
+https://www.digitalocean.com/community/tutorials/how-to-install-and-use-docker-on-ubuntu-16-04
+
+## Dockerfile description
+- Ubuntu base system (Ubuntu 20.04 by default)
```

Comment:
Given that the base image is Ubuntu20, can it still be deployed on Ubuntu 16 host?

Reply:
yes